### PR TITLE
Update main.js

### PR DESCRIPTION
**Pull request by jsagardoy**
Changes made to ensure that Houses are loaded and displayed before Characters

### CODE REVIEW

File: 01 Non blocking/README.md [hunk 1]
```diff
@@ -6,22 +6,23 @@ $ npm install
...
     * While this request `service.getHousesByName(houseInput.value, housesRequestConfig);` does not end...
     * This one `service.getCharactersByName(characterInput.value, charactersRequestConfig);` is not going to start
 
-* If we already have been working with callbacks, we are use to initialize a request and then proceed with the execution.
+* If we have been already working with callbacks, we are used to initialize a request and then proceed with the execution.
+
+* But if just one thing at a time is executed, how is this possible?
 
-* But if just one thing per time it's executed, how is this possible?
+* Inside Chrome developer tools, on the network tab, we can notice that the requests are being sent concurrently.
```
Comment (JaimeSalas) [inline 01 Non blocking/README.md]: Why remove the last phrase?

File: 01 Non blocking/README.md [hunk 2]
```diff
@@ -6,22 +6,23 @@ $ npm install
...
-$ npm install
+$ npm start
 ```
 * To get results on the open browser app:
     * house: `House Lannister of Casterly Rock`
     * character input: `Jaime Lannister`
 
 ## In this demo we are going to study the non blocking nature of JavaScript
 
-* Just one thing can happen per time in JavaScript. But the required time to get data loaded is really short. Is not just to have a really good network capabilities, there is another reason.
```
Comment (JaimeSalas) [inline 01 Non blocking/README.md]: Thanks good update.

File: 01 Non blocking/src/js/main.js [hunk 3]
```diff
@@ -9,20 +9,22 @@ document.onreadystatechange = () => {
...
-                service.getHousesByName(houseInput.value, housesRequestConfig);
-                console.log('2');
-                const charactersRequestConfig = {
-                    err: handleError,
-                    callback: handleCharactersRequestSuccess(apiMapper, printer)
-                };
-                service.getCharactersByName(characterInput.value, charactersRequestConfig);
+              service.getHousesByName(houseInput.value, housesRequestConfig);
+              setTimeout(() => {
+                  console.log('2');
```
Comment (JaimeSalas) [inline 01 Non blocking/src/js/main.js]: No setTimeout solution, it's not a good approach.

File: 01 Non blocking/README.md (hunk 1 above)
Comment (jsagardoy) [reply 01 Non blocking/README.md]: typo!

**Comment by jsagardoy** [comment]
updated files due to comments

**Pull request closed without merging**